### DISCUSSION TITLE
teoria relazionale - 7

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#74] [Esercizi]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/82e9c7cd-5c42-4299-a1f1-096457a66f3f)

**Comment by Frisbon** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/aa7f69d6-7126-48f4-85e5-368676fc507c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/84731341/3150eac6-8c99-48c8-b38f-a76c995443e8)

**Comment by 7Andrei** [comment]
Soluzione uguale a @Frisbon
